### PR TITLE
fix ShapeEntityItem collision shapes

### PR DESCRIPTION
**Pull request by AndrewMeadows**
There were two final bugs preventing ShapeEntityItem collisions shape from being correct.

### CODE REVIEW

File: libraries/render-utils/src/GeometryCache.cpp [hunk 1]
```diff
@@ -170,7 +170,7 @@ void GeometryCache::ShapeData::setupVertices(gpu::BufferPointer& vertexBuffer, c
     gpu::Buffer::Size offset = vertexBuffer->getSize();
     vertexBuffer->append(vertices);
 
-    gpu::Buffer::Size viewSize = vertices.size() * 2 * sizeof(glm::vec3);
+    gpu::Buffer::Size viewSize = vertices.size() * sizeof(glm::vec3);
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/GeometryCache.cpp:173]: This bug was my fault.

File: libraries/render-utils/src/GeometryCache.cpp [hunk 2]
```diff
@@ -150,7 +150,7 @@ void GeometryCache::computeSimpleHullPointListForShape(const int entityShape, co
 
 
         uniqueVerts.push_back(curVert);
-        outPointList.push_back(curVert * entityHalfExtents);
+        outPointList.push_back(curVert * entityExtents);
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/GeometryCache.cpp]: Turns out we want to scale by the entity dimensions rather than halfDimensions.

Comment (1P-Cusack) [reply libraries/render-utils/src/GeometryCache.cpp]: Thanks! This confirms an untested local change I had late Friday.  I wasn't sure about it though.

**Comment by AndrewMeadows** [comment]
Closing this PR.  These changes have been ported to the target branch.

**Pull request closed without merging**